client/src/components/pages/homepage/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link, BrowserRouter as Router } from 'react-router-dom';
+
+import Button from 'components/utils/Button'
+import './style.css';
+
+function Home() {
+  return (
+    <section className='home'>
+      <p className='home--desc'>
+        Lorem ipsum dolor sit amet, consectetuer adipiscing elit.
+        Aenean commodo
+      </p>
+      <div className='home--register'>
+        <Router>
+          <Link to='/login'>
+            <Button onClick={() => undefined} className='home--button' children='Login' />
```

Comment:
@IsraaSulaiman React automatically puts any child content that it finds in the jsx into the `children` prop for a component, which means that you can rewrite this to look a little more like natural html:
```
<Button onClick={() => undefined} className='home--button'>
  Login
</Button>
```